app/views/follows/_followers.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="row">
+  <div class="small-12 column">
+
+    <% if followable.follows.blank? %>
+      <div class="callout primary text-center">
+        <%= t('follows.no_followers') %>
+      </div>
+    <% end %>
```

Comment:
Adding here an `else` and including inside it the following `.each` loop could maybe be safer?